### PR TITLE
[Intl] fix nullable phpdocs and useless method visibility of internal class

### PR DESCRIPTION
**Pull request by Tobion**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no 
| BC breaks?    | no     
| Deprecations? | no 
| Tests pass?   | yes    
| Fixed tickets |
| License       | MIT
| Doc PR        | 

Fix stuff found in #32525

### CODE REVIEW

File: src/Symfony/Component/Intl/DateFormatter/DateFormat/FullTransformer.php [hunk 1]
```diff
@@ -69,16 +69,6 @@ public function __construct($pattern, $timezone)
         ];
     }
 
-    /**
-     * Return the array of Transformer objects.
-     *
-     * @return Transformer[] Associative array of Transformer objects (format char => Transformer)
-     */
-    public function getTransformers()
```
Comment (Tobion) [inline src/Symfony/Component/Intl/DateFormatter/DateFormat/FullTransformer.php]: This is an internal class

**Review by chalasr — APPROVED** [review]

File: src/Symfony/Component/Intl/DateFormatter/DateFormat/FullTransformer.php (hunk 1 above)
Comment (nicolas-grekas) [reply src/Symfony/Component/Intl/DateFormatter/DateFormat/FullTransformer.php]: I think we should change this class on 4.4
I get it's internal, but let's keep 3.4 super-unstable.

**Comment by nicolas-grekas** [comment]
Thank you @Tobion.

File: src/Symfony/Component/Intl/DateFormatter/DateFormat/FullTransformer.php (hunk 1 above)
Comment (nicolas-grekas) [reply src/Symfony/Component/Intl/DateFormatter/DateFormat/FullTransformer.php]: change reverted while merging

Comment (Tobion) [reply src/Symfony/Component/Intl/DateFormatter/DateFormat/FullTransformer.php]: #33056

**Pull request merged**